html_css/implementing_accessibility.md

```diff
@@ -0,0 +1,221 @@
+Now that you know to take the contrast ratio into account when adding color for text and backgrounds, let's see if you can tell which of the buttons in the image below is red:
+
+![Color blindness simulation](https://user-images.githubusercontent.com/70952936/125673910-66b43803-3228-4920-98f8-80ac063ef344.jpg)
+
+The answer, by the way, is button 4. The image above is simulating achromatopsia, which is also known as total color blindness. This illustrates a key point when using color: **you shouldn't use color alone to convey information**. There may be exceptions where it's necessary to only use color, but generally you want to follow this rule. 
+
+Let's look at another example. Imagine a form that has an instruction stating that required fields are indicated by red text. If a user is color blind or otherwise has difficulty telling some colors apart, having to rely solely on the color of this text can make it difficult or even impossible to perceive or operate the page. Instead of only using color, a form could indicate all required fields with red text and an asterisk (though this isn't the only way to convey that a form input is required).
+
+![Conveying information with color](https://user-images.githubusercontent.com/70952936/125674026-9baafc58-2339-48f4-8b12-892375b87ad7.jpg)
+
+#### Themes
+
+We're not going to go in depth when it comes to themes since the topic can get pretty deep, but there are a couple of things that you should know:
+
+1. A dark theme isn't just an aesthetic option. Some users have a sensitivity to light, which can make pages with a light theme (often the default or only option) very difficult, if not impossible, to browse. There are general work-arounds such as the Dark Reader extension for both Chrome and Firefox, but such general extensions may not always be a perfect fix. Some areas of a page might not be affected by such extensions, the extension might not change the colors on the page properly (resulting in contrast issues), or the extension could interfere with the page in unforeseen ways.
+
+2. A dark theme isn't just taking the colors from the light theme and inverting them. Not only is this not 100% reliable in terms of color contrasts, but while light themes use shadows to simulate elevation for elements, dark themes instead rely on giving elements with a higher elevation a lighter background color. If you simply inverted the light theme colors, then the dark theme would have elements with white shadows, which doesn't simulate the elevation effect you should be going for (and it wouldn't really look good).
+
+### Keyboard Navigation
+
+Some users aren't able to use a mouse to navigate or operate a page and instead rely on using a keyboard or another assistive technology that can simulate keyboard inputs. Other users may even just prefer using a keyboard over a mouse, or may use a mix of both. These users require proper keyboard navigation, something that can easily be overlooked when developing a website.
```

Comment:
A question that would be raised here is: "What is proper keyboard navigation?"

Other sections are rather descriptive or have an example, can we come up with a tangible example of this to better explain the issue?